gatsby/src/components/ToC/index.js

```diff
@@ -1,36 +1,46 @@
 import React from "react";
+import styled from "styled-components";
+import { Anchor as GrommetAnchor } from "grommet";
+
+import { List, Item } from "../List";
+
+const Anchor = styled(GrommetAnchor)`
+  line-height: 1.5;
+  font-size: 12px;
+  margin-bottom: 6px;
+`;
 
 const ToC = ({ content }) => {
```

Comment:
I think naming the component TableOfContents makes it easier to understand. Please try to avoid abbreviations(ToC = Thor On Crack :)) )

Reply:
ahaha Thor on Crack thats a sight to behold.